src/Pcre/PcreUtils.php

```diff
@@ -15,6 +15,7 @@ class PcreUtils
      * @see https://www.php.net/manual/en/function.preg-match.php
      *
      * @param array<string> $matches
+     * @param 0|256|512|768 $flags
```

Comment:
File updated to match phpstan complaints

Reply:
We may be able to use the `int-mask-of<...>` type here

Reply:
stylé !